data/src/client.rs

```diff
@@ -312,6 +471,25 @@ impl Client {
                     }
                     if contains("batch") {
                         requested.push("batch");
+
+                        // We require batch for our chathistory support
+                        let requesting_chathistory = if contains_starting_with("chathistory") {
+                            requested.push("chathistory");
+                            true
+                        } else if contains_starting_with("draft/chathistory") {
+                            requested.push("draft/chathistory");
+                            true
+                        } else {
+                            false
+                        };
```

Comment:
Why `contains_starting_with` when we then insert is as the full value? We need to ensure we're requesting the exact same capability that we checked existed. Why doesn't just `contains ` work here since we check both w/ and w/out `draft` prefix?

Reply:
I thought I had run into a server that listed the `chathistory` capability with a value attached (e.g. `draft/chathistory=1000`), and I used `contains_starting_with` to sidestep having an issue with it, but I can't find any server where that's the case now.  Will switch it back to `contains` since that's my reading of the specification.